drivers/leds/leds-ktd202x.c

```diff
@@ -0,0 +1,417 @@
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
```suggestion
// SPDX-License-Identifier: GPL-2.0-or-later
```

(see https://spdx.org/licenses/#deprecated)